modules/apps/questions/questions-web/src/main/java/com/liferay/questions/web/internal/portlet/QuestionsPortlet.java

```diff
@@ -49,20 +47,16 @@
 import java.util.Collections;
 import java.util.Comparator;
 import java.util.List;
-import java.util.Map;
 import java.util.Objects;
 import java.util.Properties;
 
-import org.osgi.service.component.annotations.Activate;
 import org.osgi.service.component.annotations.Component;
-import org.osgi.service.component.annotations.Modified;
 import org.osgi.service.component.annotations.Reference;
 
 /**
  * @author Javier Gamarra
  */
 @Component(
-	configurationPid = "com.liferay.questions.web.internal.configuration.QuestionsConfiguration",
```

Comment:
This isn't really unnecessary. If you look for usages of the configuration interface, it is used in one of the JSPs. The issue here is that they're reading the configuration again in a JSP instead of using the one referenced in the portlet.

But it is true that, as the portlet will no longer wait for the configuration, it will be still available when the config. is refreshed. The issue is that now, if the configuration is not available, there will be a runtime failure.

Reply:
As we have internally talked, I've made some tests and it doesn't seem to rise a runtime failure. So, I will forward it. Thanks